src/common/com/intellij/plugins/haxe/model/fixer/HaxeCreateMethodFixer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2000-2013 JetBrains s.r.o.
+ * Copyright 2014-2015 AS3Boyan
+ * Copyright 2014-2014 Elias Ku
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.plugins.haxe.model.fixer;
+
+import com.intellij.plugins.haxe.lang.psi.HaxeReferenceExpression;
+import com.intellij.psi.PsiElement;
+
+public class HaxeCreateMethodFixer extends HaxeFixer {
+  public HaxeCreateMethodFixer(String name, PsiElement context) {
+    super("Create method");
+  }
+
+  @Override
+  public void run() {
+
```

Comment:
I realize these are just stubs and that you aren't using them yet.   At least place a TODO in them so that they're easily locatable.  Or have them log "unimplemented".